cmd/kube-apiserver/app/server.go

```diff
@@ -547,6 +553,17 @@ func BuildAuthorizer(s *options.ServerRunOptions, versionedInformers clientgoinf
 	return authorizationConfig.New()
 }
 
+// BuildRequestManager constructs the request manager
+func BuildRequestManager(s *options.ServerRunOptions, extclient clientgoclientset.Interface, versionedInformer clientgoinformers.SharedInformerFactory) utilflowcontrol.Interface {
+	return utilflowcontrol.NewRequestManager(
+		versionedInformer,
+		extclient.FlowcontrolV1alpha1(),
+		s.GenericServerRunOptions.MaxRequestsInFlight+s.GenericServerRunOptions.MaxMutatingRequestsInFlight,
+		s.GenericServerRunOptions.RequestTimeout/4,
```

Comment:
this looks weird. Comment explaining why dividing by four makes sense, even if it's just "because it seemed like a good idea".  Otherwise it will end up enshrined.

Reply:
Yes, this deserves documentation.  I could imagine a pointer into the KEP, a pointer into the user-facing doc (which we have barely started drafting), or some commentary here.

The reasoning I have so far is as follows.  This is the timeout for waiting in a queue.  It needs to be shorter than the total timeou, which is `s.GenericServerRunOptions.RequestTimeout`.  I think it should be a definite minority of the overall timeout, because we want to mostly reserve that for request execution.  Also because keeping things for a long time in queues is bad.  25% of the overall timeot is my initial guess at a good fraction.

Reply:
yeah, we need abundant comment/explanation for the magic number here, i will put it in detail..